hack/snapshot.sh

```diff
@@ -33,5 +33,6 @@ authenticate
 buildImages $HELM_CHART_VERSION
 cosignImages
 publishHelmChart
-notifyRelease "snapshot" $HELM_CHART_VERSION
 notifyIfStableRelease
+notifyRelease "snapshot" $HELM_CHART_VERSION
+pullPrivateReplica "snapshot" $SNAPSHOT_TAG
```

Comment:
should pullPrivateReplica happen before the notify steps so that we verify a pull and eventually push to private repos before notifying?

Reply:
Yes!